### ISSUE TITLE
why in your config, that _C.TRAIN.PRETRAINING.FRAME_PADDING is 0. Shouldn't it be 1?

### ISSUE THREAD

**Issue by yechenzhi** [csiro-robotics/TCE#4]
https://github.com/csiro-robotics/TCE/blob/55e76ceba5d400c1698df3fb3ee072226d5a12d8/config/default.py#L99

https://github.com/csiro-robotics/TCE/blob/55e76ceba5d400c1698df3fb3ee072226d5a12d8/datasets/pretraining/kinetics400.py#L18

https://github.com/csiro-robotics/TCE/blob/55e76ceba5d400c1698df3fb3ee072226d5a12d8/datasets/pretraining/kinetics400.py#L27

**Comment by JBKnights** [comment]
Hello!  So the reason for this is that we defined frame padding as additional padding between frames; consequently, the default value is 0.

**Comment by yechenzhi** [comment]
Shouldn't **pair_idx** be the anchor_frame's **next frame**, as you claimed in your paper?  

And one more question, in Kinetics dataset, you said you only use one anchor_frame for each pre-training video,  is this enough to get temporally coherent embeddings?

**Comment by JBKnights** [comment]
Hello,

It would appear that you are correct - frame padding should equal one!  Apologies, likely an error when we were re-writing the code for release; we'll push a fix ASAP.  

As for your second question, we found that overall our network tended to produce temporally coherent TSNEs with this training method; we are satisfied that it was sufficient to form temporally coherent embeddings.  While our paper didn't explore the possibility of leveraging multiple anchors, it's very possible an approach down these lines could result in improved coherency.
